### PR TITLE
Drop removed package and enable AST rules for templates

### PR DESCRIPTION
**Pull request by bartektelec**
Removed eslint-plugin-airbnb-base because it is just a placeholder package
image

### CODE REVIEW

**Comment by bartektelec** [comment]
Moved to eslint-plugin-svelte to get more control over HTML template rules. Bound the version as the plugin and parser are still in experimental versions.
Added rules for templates:
- Error when no explicit `[type]` attribute in `<button>` - could cause unexpected form submit
- Error when unnecessary interpolation mustache syntax i.e. `{"Hello"}`
- Error when not using shorthand attribute i.e. `disabled={disabled}`
- Warning when using `[target="_blank"]` without `[rel="noopener noreferrer"]` because of XSS vuln


image

File: svelte/index.js [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
+      parser: 'svelte-eslint-parser',
+      parserOptions: {
+        parser: '@typescript-eslint/parser'
+      }
     }
   ],
   rules: {
+    '@ota-meshi/svelte/no-at-html-tags': 1,
+    '@ota-meshi/svelte/no-target-blank': 1, // can cause XSS attack
+    '@ota-meshi/svelte/shorthand-attribute': 2,
```
Comment (Fedoskina) [inline svelte/index.js:20]: you are tough but OK:)
What do you think about adding the check for unused imports?
https://www.npmjs.com/package/eslint-plugin-unused-imports

**Review by neonwarp — APPROVED** [review]

File: svelte/index.js (hunk 1 above)
Comment (bartektelec) [reply svelte/index.js:20]: We already have a similar rule that sort of does the same thing:

image

Comment (Fedoskina) [reply svelte/index.js:20]: Hm, I haven't had it, will check my eslint

**Pull request merged**